Description: Replace sage-python23 by python
 The command sage-python23 is intended to run sage's own python.
 In our case we only have the system python.
Author: Tobias Hansen <thansen@debian.org>
Bug-Debian: https://bugs.debian.org/902545

--- a/sage/src/bin/sage
+++ b/sage/src/bin/sage
@@ -607,7 +607,7 @@
 
 if [ "$1" = "-docbuild" -o "$1" = "--docbuild" ]; then
     shift
-    exec sage-python23 -m "sage_setup.docbuild" "$@"
+    exec python -m "sage_setup.docbuild" "$@"
 fi
 
 if [ "$1" = '-gdb' -o "$1" = "--gdb" ]; then
@@ -617,7 +617,7 @@
         gdb -x "$SAGE_SCRIPTS_DIR/sage-gdb-commands" \
             -args python "$SAGE_SCRIPTS_DIR/sage-ipython" "$@" -i
     else
-        sage_dir=$(sage-python23 -c 'import os, sage; print(os.path.dirname(sage.__file__))')
+        sage_dir=$(python -c 'import os, sage; print(os.path.dirname(sage.__file__))')
         cygdb "$sage_dir" "$SAGE_SRC/sage" \
             -- -x "$SAGE_SCRIPTS_DIR/sage-gdb-commands" \
             -args python "$SAGE_SCRIPTS_DIR/sage-ipython" "$@" -i
--- a/sage/src/bin/sage-eval
+++ b/sage/src/bin/sage-eval
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 
 import sys
 from sage.all import *
--- a/sage/src/bin/sage-fixdoctests
+++ b/sage/src/bin/sage-fixdoctests
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 
 """
 Given the output of doctest and a file, adjust the doctests so they won't fail.
--- a/sage/src/bin/sage-ipython
+++ b/sage/src/bin/sage-ipython
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
 Sage IPython startup script.
--- a/sage/src/bin/sage-list-packages
+++ b/sage/src/bin/sage-list-packages
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 r"""
 Script to list the Sage packages
 
--- a/sage/src/bin/sage-preparse
+++ b/sage/src/bin/sage-preparse
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 """
 Preparse .sage files and save the result to .sage.py files.
 
--- a/sage/src/bin/sage-run-cython
+++ b/sage/src/bin/sage-run-cython
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 
 import sys
 from sage.repl.load import load_cython
--- a/sage/src/bin/sage-runtests
+++ b/sage/src/bin/sage-runtests
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 
 import optparse
 import os
--- a/sage/src/bin/sage-startuptime.py
+++ b/sage/src/bin/sage-startuptime.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env sage-python23
+#!/usr/bin/env python
 
 ########################################################################
 # Originally based on a script by Andrew Dalke:
